pkg/ccl/crosscluster/logical/logical_replication_writer_processor.go

```diff
@@ -491,7 +501,16 @@ func (lrw *logicalReplicationWriterProcessor) flushBuffer(
 				if err != nil {
 					// TODO(ssd): Handle errors. We should perhaps split the batch and retry a portion of the batch.
 					// If that fails, send the failed application to the dead-letter-queue.
-					return err
+
+					// TODO(azhu): Remove these nil variables once refactoring is done
+					// & we've implemented conflict handling logic.
+					var kv roachpb.KeyValue
+					var cdcEventRow cdcevent.Row
+					var writeTimestamp hlc.Timestamp
+					var conflictReason string
```

Comment:
the conflict reason could just be err.Error() for now.

Reply:
For this comment and the one below - I ended up removing this block of code entirely since we'll have lrw.dlq call dlqClient.log() given the changes in https://github.com/cockroachdb/cockroach/pull/126525